.changeset/lucky-garlics-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/input-option': major
```

Comment:
need changesets for other impacted components

Reply:
This is only being used in SearchInput so far. I'll open another PR that refactors Combo/Select

Reply:
Looks like we're impacting files in lib, palette, polymorphic too

Reply:
Yep. Those all have separate changeset messages